charts/apisix/templates/configmap.yaml

```diff
@@ -326,6 +326,8 @@ data:
           - name: "admin"
             {{- if .Values.apisix.admin.credentials.secretName }}
             key: ${{"{{"}}APISIX_ADMIN_KEY{{"}}"}}
+            {{- else if eq .Values.apisix.admin.credentials.admin "" }}
+            key: {{ randAlphaNum 32 }}
```

Comment:
How do users know the key? You need to add a prompt output.

Perhaps managing admin-key through a secret would be more appropriate.
What do you think？

Reply:
Since this is in the confimap. Users can look at the configmap using kubectl to look at the key. Is there an issue there?

Reply:
I think we can write how to get the key in the docs.

Reply:
I have added that in the README against this particular key. https://github.com/apache/apisix-helm-chart/pull/740/files#diff-3169e4e147b847e17edc927d65d4aa9cb5db1a436d62acf059c166e898f9f3fdR46

Reply:
You can add a prompt here.
https://github.com/apache/apisix-helm-chart/blob/master/charts/apisix/templates/NOTES.txt

Reply:
Is it regenerated when upgrading the version?